components/hero/hero.sass

```diff
@@ -1,22 +1,16 @@
 .hero
   position: relative
-  height: 100vh
+  height: 85vh
   width: 100%
   display: flex
   align-items: center
   justify-content: center
 
-.hero__background
-  background-image: url('https://images.unsplash.com/photo-1550847067-93887e03cfbe?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=3000&q=80')
+.hero__logo
+  background-image: url('/static/images/we-mountains-color.png')
   background-size: cover
   background-repeat: no-repeat
   background-position: center
   position: absolute
-  height: 100%
-  width: 100%
-  z-index: -1
-
-.hero__header
-    font-size: 5rem
-    font-weight: 400
-    color: white
\ No newline at end of file
+  +size-square(40%)
```

Comment:
Forgot to mention this, make sure you explicitly @import your mixins.

That way if we ever wind up with two mixins that have the same name, you're sure to have the right one in each file :)

Reply:
Hey, not sure what you mean... Can you show me an example of what it would look like and what file the @import would go in?

Reply:
@import ../../globals/sass-config/_mixins

This is useful if our _mixins.sass file gets too big and we ever have to split it into things like
_typography-mixins.sass
_positioning-mixins.sass

This is something I've had to do before, I have one repo w/ about 20 mixin files. Because mixins are rad and if you have to do something more than a few times, it's worth making a quick mixin :)

Reply:
Gotchya! Thanks for the explanation